Stripe/Payments/STPThreeDSTextFieldCustomization.h

```diff
@@ -0,0 +1,43 @@
+//
+//  STPThreeDSTextFieldCustomization.h
+//  StripeiOS
+//
+//  Created by Yuki Tokuhiro on 6/18/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+@import UIKit;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A customization object to use to configure the UI of a text field.
+ */
+@interface STPThreeDSTextFieldCustomization : NSObject
+
+/// The default settings.
++ (instancetype)defaultSettings;
+
+/// The border width of the text field. Defaults to 2.
```

Comment:
is this points or pixels? Should it be a CGFloat instead?

Reply:
Ah I missed this one, should be CGFloat for sure.  Will follow up.